saskatoon/sitebase/templates/app/base/header.html

```diff
@@ -31,7 +32,7 @@ <h2 style="color:white">Les Fruits Défendus</h2>
                                     <li><a href={% url "logout" %}>{% translate "Logout" %}</a></li>
                                 </ul>
                             </li>
-                            {% if user.is_staff %}
```

Comment:
Actually pickleaders are currently still marked as staff=True on our prod database and have access to things related to harvests (see pickleader group permissions).
So I would use `is_core_or_admin and is_staff` instead of `or`

Reply:
ok let's keep `is_staff` for the whole dropdown. Additionally we could add a check `{% if user|has_group:'admin' or user|has_group:'core' %} on the specific manage translations <li> item.